actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Include source location in routes extended view.
```

Comment:
Can we include example of this change from original PR?

Reply:
Updated 😄 thanks!